### ISSUE TITLE
Install gpu-operator on network restricted environment.

### ISSUE THREAD

**Issue by kousui-dev** [NVIDIA/gpu-operator#427]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._

### 1. Quick Debug Checklist
- [ ] Are you running on an Ubuntu 18.04 node?:No
- [ ] Are you running Kubernetes v1.13+?:Yes(OCP4.7.51, 4.8.41, 4.10.26)
- [ ] Are you running Docker (>= 18.06) or CRIO (>= 1.13+)?:CRIO

### 1. Issue or feature description
There was a problem when installing GPU-operator 1.8.2 in an environment where NW was restricted (it is possible to go out to the Internet, but the NW band is very narrow).
The GPU-operator image was placed in the mirror repository, but when the GPU-operator was installed, 88 worker nodes obtained materials via the Internet, resulting in a timeout and the Pod not starting normally.
As a result of the analysis, it was found that the material was acquired externally by yum/dnf.
When I contacted Redhat, they advised me to create a "Yum mirror" and configure the driver container as follows.
https://cloud.redhat.com/blog/how-to-deploy-operators-on-disconnected-environments

I am planning to reinstall GPU-operator 1.10.1 and update to 1.11.0.
So here's the question.

1. If I prepare the above "Yum mirror" when reinstalling 1.10.1 and when updating to 1.11.0, will it not be acquired externally?

2. Is it okay to include only the following packages listed in the URL above for "Yum mirror"? (The article is old, but I would like to know if the 1.10.1 and 1.11.0 are the same with older version)
- elfutils-libelf.${HOST_ARCH}
- elfutils-libelf-devel.${HOST_ARCH}
- kernel-headers-${GPU_NODE_KERNEL_VERSION}
- kernel-devel-${GPU_NODE_KERNEL_VERSION}
-kernel-core-${GPU_NODE_KERNEL_VERSION}
If there are other packages I need, please let me know the list of what I need.

3. How can I check the above ${HOST_ARCH}, ${GPU_NODE_KERNEL_VERSION}? (The article said that you can check it with "oc describe node", but I couldn't check it.)

### 2. Steps to reproduce the issue
installing gpu-operator on network restricted environment.

### 3. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)
There are no attached file.

**Comment by shivamerla** [comment]
@kousui-dev Please refer to [this](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/mirror-gpu-ocp-disconnected.html) guide on using with restricted networks. Network connectivity is required mainly to compile and install NVIDIA drivers and starting from `v1.9.0` we have avoided this dependency with OCP 4.9.9+ using Driver-Toolkit images(provided by RH) for driver installation. So at-least on OCP versions >4.9.9 mirroring of packages is not required. I see that you are using 4.7 and 4.8 OCP versions where this is still required.

**Comment by shivamerla** [comment]
Also, note that on OCP 4.9+ versions, latest GPU operator is `v22.9.0` with versioning scheme change. Upgrades from previous versions are supported.

**Comment by kousui-dev** [comment]
Thank you for your reply.
To share the current situation, OCP is currently being upgraded.
When gpu-operator1.8.2 was installed on OCP4.7.51, the above phenomenon (materials acquired externally) occurred.
I'm planning to upgrade OCP to 4.10.26, reinstall gpu-operator1.10.1, and upgrade to gpu-operator1.11.0.
(Because verification with OCP4.10.26 and gpu-operator1.11.0 has been completed, I would like to postpone the introduction of v22.9.0)

Just to be sure, I recognized that yum mirror is not necessary when reinstalling gpu-operator1.10.1 or upgrading to gpu-operator1.11.0 for OCP4.10.26, is this correct?

**Comment by shivamerla** [comment]
Yes, mirror repository is not required with those versions as we support entitlement free builds.

**Comment by rajathagasthya** [comment]
Resolving this issue, as the necessary information was provided.
